### ISSUE TITLE
unable to set kvm -smp cores/thrads

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#378]
Originally reported of Google Code with ID 322.

```
Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

Software version: 2.6.1
Internode protocol: 2060000
Configuration format: 2060000
OS api version: 20
Export interface: 0

<b>What distribution are you using?</b>
ubuntu 12.04 Server

<b>What steps will reproduce the problem?</b>

Windows 7 Professional supports only 2 sockets, but many cores/thrads per socket, so i like to set some more cores/threads for the CPU like:

gnt-instance modify -B vcpus="2,cores=2,threads=2" win-7-01.virtual

but i get the error "Parameter Error: Unknown parameter 'cores'"


<b>What is the expected output? What do you see instead?</b>

i like to see the -smp otione are set the way that the kvm manual shows

###################
       -smp n[,cores=cores][,threads=threads][,sockets=sockets][,maxcpus=maxcpus]
           Simulate an SMP system with n CPUs. On the PC target, up to 255 CPUs are supported. On Sparc32 target, Linux limits the
           number of usable CPUs to 4.  For the PC target, the number of cores per socket, the number of threads per cores and the
           total number of sockets can be specified. Missing values will be computed. If any on the three values is given, the total
           number of CPUs n can be omitted. maxcpus specifies the maximum number of hotpluggable CPUs.
###################

<b>Please provide any additional information below.</b>

```
Originally added on 2012-11-23 13:17:03 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 16:44:09 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 16:44:26 +0000 UTC.

Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 16:45:15 +0000 UTC.

Added Labels: Hypervisor-KVM

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-19 13:21:47 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed in:

4888a60 kvm: allow setting smp cores, threads, sockets

```
Originally added on 2013-02-26 15:58:26 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
